pkg/csi/service/common/constants.go

```diff
@@ -112,12 +112,34 @@ const (
 	// VsanDatastoreType is the string to identify datastore type as vsan.
 	VsanDatastoreType string = "vsan"
 
-	// AllowRoot decides Whether the root access should be allowed on a file volume
-	AllowRoot string = "allowroot"
+	// CSIMigrationParams helps identify if volume creation is requested by
+	// in-tree storageclass or CSI storageclass
+	CSIMigrationParams = "csimigration"
 
-	// Permission to be set in the file volume
-	Permission string = "permission"
+	// AttributeInitialVolumeFilepath represents the path of volume where volume is created
+	AttributeInitialVolumeFilepath = "initialvolumefilepath"
 
-	// IPs is Client IP address, IP range or IP subnet
-	IPs string = "ips"
+	// DatastoreMigrationParam is used to supply datastore name for Volume provisioning
+	DatastoreMigrationParam = "datastore"
```

Comment:
Can you update these based on you latest PR https://github.com/kubernetes/kubernetes/pull/90911?

Reply:
done. https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/235/commits/4376eaa2126b2741e10c4800ca60ae1032b486a7